indigo/indigo/src/main/scala/indigo/shared/platform/SceneProcessor.scala

```diff
@@ -116,17 +117,27 @@ final class SceneProcessor(
 
     val displayLayers: scalajs.js.Array[(DisplayLayer, scalajs.js.Array[(String, DisplayObject)])] =
       scene.layers
-        .flatMap(_.toBatch)
+        .flatMap(l =>
+          l.toBatch.map(
+            (
+              l match {
+                case LayerEntry.Tagged(tag, _) => Some(tag)
+                case _                         => None
+              },
+              _
+            )
+          )
+        )
```

Comment:
Is it not desirable to do this after the visible check?

Reply:
We can't because at the pointer we're at `scene.layers` we have a `LayerEntry`, which doesn't contain the `visible` check.. this is only available after we convert the `LayerEntry` to `Content` in the `toBatch` method of the `LayerEntry`.
We could, of course, filter the `toBatch` method... so I might do that